libs/island-ui/core/src/lib/Input/types.ts

```diff
@@ -52,6 +52,7 @@ export interface InputComponentProps {
     on: boolean
     maxHeight?: number
   }
+  inputMode?: string
```

Comment:
Just a thought, do we want to define a list of possible values for this (i.e. inputMode: "numeric" | "decimal" | ...) or is that not a good idea because we'd have to update our code if this attribute ever changes 🤔 

What do you think?

Reply:
That would be better for the consumer of the component, but I always try to mirror the HTML element typings, so I put just a `string`. But I just looked back at it and it turns out that I was reading the `interface HTMLInputElement {}` from Typescript and not `interface HTMLAttributes {}` from React. The element uses the React typings and they have the values specified. 
So I'm going to change it. It was a good thing that you flagged this.